app/api/currentuser/get_group_invitations.go

```diff
@@ -110,10 +110,16 @@ type groupWithApprovals struct {
 func (srv *Service) getGroupInvitations(w http.ResponseWriter, r *http.Request) service.APIError {
 	user := srv.GetUser(r)
 
-	query := srv.GetStore(r).GroupMembershipChanges().
+	query := srv.GetStore(r).
+		Table(`
+			(SELECT group_id, member_id, MAX(at) AS at, action, ANY_VALUE(initiator_id) AS initiator_id
```

Comment:
Do we really don't care which of possible `initiator_id` is chosen within rows having same (group_id, member_id, action)?

Reply:
I had the same remark.
Possibly there could be in practice a single invitation for a same user-group, but if that's the case, the condition is still a bit weirdy